expressjs/src/routes/v1/System.ts

```diff
@@ -18,6 +20,19 @@
 
 // -- Routes -- //
 
+// Provide the local config.yml file to the UI
+router.get('/v1/system/config_yml', (_req, res) => {
+  const ymlPath = '/app/config.yml'
+
+  // Check for and read the config.yml file
+  if (fse.existsSync(ymlPath)) {
+    const ymlConfig = yaml.load(fse.readFileSync(ymlPath, 'utf8'))
+    return res.json(ymlConfig)
+  }
+
+  return res.json({ local_config: false })
+})
```

Comment:
Opted for using express-slow-down instead which applies slow down rather than rate limiting, preventing abuse but without adding a limit on requests.